projets.html

```diff
@@ -0,0 +1,97 @@
+                </ol>
+            </nav>
+            <p class="display-2 fw-semibold mt-4 inkFreeFamily">Mes projets</p>
+        </div>
+        <div class="container">
+            <div class="row justify-content-center mt-4">
+                <div class="col-11">
+                    <h4 class="mt-3 mb-3 text-center">Jeu de mémoire - Javascript</h4>
+                    <div class="card">
+                        <img src="assets/memoire_index.png" width="auto" height="auto" class="shadow">
+                        <div class="card-body">
+                            <p class="card-text text-justify p-3">Le jeu de mémoire est un jeu basé sur le html5 et qui peut être joué sur n'importe quel appareil doté d'un navigateur web. Il utilise javascript et css3 pour créer une expérience amusante et divertissante. L'utilisateur choisi sont nombre de paires et le but du jeu est de faire correspondre ses paires de cartes. Si vous trouvez toutes les paires dans le temps alloué, vous gagnez la partie. Si le temps est écoulé et qu'il reste des cartes à découvrir, vous perdez la partie.</p>
+                        </div>
+                    </div>
+                    <h4 class="mt-5 mb-3 text-center">Maquette graphique - Figma</h4>
+                    <div class="card bg-dark text-white">
+                        <img src="assets/projet4.png" width="auto" height="auto" style="box-shadow: 0 0 10px rgba(255,255,255,0.5)">
+                        <div class="card-body">
+                            <p class="card-text text-justify p-3">Une maquette graphique est une étape essentielle dans la création d'un site e-commerce, car elle permet de visualiser en détail l'apparence et l'interface du site. Découvrez comment créer une maquette graphique efficace pour votre site e-commerce.</p>
+                        </div>
+                    </div>
+                    <h4 class="mt-5 mb-3 text-center">Portfolio - NodeJs + EJS</h4>
+                    <div class="card">
+                        <img src="assets/portfolio.png" width="auto" height="auto" class="shadow">
+                        <div class="card-body">
+                            <p class="card-text text-justify p-3">Mon portfolio web est un site internet professionnel qui me permet de présenter mon parcours, mes compétences et mes réalisations de manière claire et concise. Il s'adresse à un public varié, tel que des employeurs potentiels, des clients, des partenaires ou des collaborateurs. Il contient une page de présentation , une page de mes projets réalisés et une page avec un formulaire pour me joindre par courriel.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>    
+        </div>
+    </main>
+    <footer class="footer-basic">
+        <div class="social">
+            <a href="https://github.com/NateThibault"><img src="assets/github.svg" width="30px" height="30px"></a>

```

Comment:
Try to use `target='_blank'` instead to open in the same page. The same for the next external links.

Reply:
Les corrections ont été appliquées